api/pom.xml

```diff
@@ -17,6 +17,11 @@
          <artifactId>mutiny</artifactId>
          <optional>true</optional>
       </dependency>
+      <dependency>
+         <groupId>org.hibernate.search</groupId>
+         <artifactId>hibernate-search-mapper-pojo-base</artifactId>
```

Comment:
wasn't supposed the new annotation to be independent of HSearch annotations?

Reply:
Great question! Theoretically you shouldn't have any Hibernate Search 6 dependencies on our own annotations, I totally agree with that.
The real problem here is that at the moment the only way to get the annotations processed by Hibernate Search is to annotate them with `@PropertyMapping` or `@TypeMapping` or ...
I had a chat with @yrodiere a while ago and without a significant investment on Search that's the only way to do that.
In my opinion we should accept this compromise and eventually we can remove it when Search is ready for it.
In the meantime I'm adding a note on top of this dependency.

Reply:
FWIW, the discussion: https://hibernate.zulipchat.com/#narrow/stream/132092-hibernate-search-dev/topic/custom.20annotations